src/main/resources/lib/guillotine/query/content-api.js

```diff
@@ -222,32 +223,32 @@ function createContentApiType(context) {
     });
 }
 
-function getContentKey(env, context) {
-    if (context.isGlobalMode()) {
-        if (env.context && env.context['__siteKey']) {
-            return wildcardLib.replaceSitePath(env.args.key, getSiteLib.getSiteFromQueryContext(env.context)._path);
-        }
+function getContentByKey(key, context, returnRootContent) {
+    const content = contentLib.get({
+        key: key
+    });
+    if (content && content._path === '/' && returnRootContent === false) {
         return null;
-    } else {
-        return wildcardLib.replaceSitePath(env.args.key, portalLib.getSite()._path);
     }
+    return securityLib.filterForbiddenContent(content, context);
 }
 
-function getContent(env, context) {
+function getContent(env, context, returnRootContent) {
     if (env.args.key) {
-        const key = getContentKey(env, context);
-        if (!key) {
-            return null;
-        }
-        const content = contentLib.get({
-            key: key
-        });
-        return content && securityLib.filterForbiddenContent(content, context);
+        const site = context.isGlobalMode() && env.context && env.context['__siteKey']
+                     ? getSiteLib.getSiteFromQueryContext(env.context)
+                     : portalLib.getSite();
+
+        const key = site ? wildcardLib.replaceSitePath(env.args.key, site._path) : env.args.key;
```

Comment:
Are we using a relative path?
If so - then it won't work for cases when key is an id